server/models/dashboard.js

```diff
@@ -36,6 +36,7 @@ const boxesSchema = Joi.array().items(
       temperature_use_custom_value: Joi.boolean(),
       temperature_min: Joi.number(),
       temperature_max: Joi.number(),
+      colors: Joi.array().items(Joi.string().allow(null)),
```

Comment:
Why does a color could be null?

Reply:
In my first occurence (before your feedbacks), an user could let values empty, to fallback on default ones.
But now, default values are set in forms, so we don't need a nullable field anymore. 👌